src/ESNDavImportClient.js

```diff
@@ -0,0 +1,21 @@
+import { davImportApi, Client } from 'esn-api-client/src';
+
+export default class ESNDavImportClient {
+  constructor(uploadFile) {
+    if (!uploadFile || typeof uploadFile !== 'function') {
```

Comment:
Isn't this enough already?

```suggestion
    if (typeof uploadFile !== 'function') {
```

Reply:
Done